proc/threads.c

```diff
@@ -1944,7 +1944,7 @@ int proc_threadsList(int n, threadinfo_t *info)
 		}
 		else {
 			map = threads_common.kmap;
-			hal_memcpy(info[i].name, "[idle]", sizeof("[idle]"));
+			hal_memcpy(info[i].name, "[kthread]", sizeof("[kthread]"));
```

Comment:
Why, kernel threads just can't have its names ? eg. `[idle]`, `[klog]`.

Reply:
Yes, I see, but asked why, it is (and woud be) a hack anyways.
`[kthread]` eating 99% of the  cpu looks weird (I liked `[idle]`).

Reply:
I get too many question lately "why there're two idle threads on single core system?" :)
This PR is just a proposition, to @Darchiv to decided whether we want this or not

Reply:
![2024-06-05-095634_936x182_scrot](https://github.com/phoenix-rtos/phoenix-rtos-kernel/assets/141153/7fd858be-74b9-4ab0-b816-788d6373b680)